attachments/09_shader_modules.cpp

```diff
@@ -60,7 +60,8 @@ class HelloTriangleApplication
 	    vk::KHRSwapchainExtensionName,
 	    vk::KHRSpirv14ExtensionName,
 	    vk::KHRSynchronization2ExtensionName,
-	    vk::KHRCreateRenderpass2ExtensionName};
+	    vk::KHRCreateRenderpass2ExtensionName,
```

Comment:
Only `VK_KHR_shader_draw_parameters` should be required. I don't think we need renderpass2 as that's core since 1.2 and we use 1.4.

Reply:
Sry my bad. If it was there, just leave it in place. I'm addressing extensions in #260